docs/features/LOGGING.md

```diff
@@ -1,13 +1,15 @@
 # Logging
 
+## NOTE: The logging features outlined in this document are to be included in version 0.3.0 and newer builds of Open Crawler.
+
 ## Configure your Crawler YAML to write system and/or event logs to file
 By default, Crawler will output system logs to `stdout`. You have the option to write these logs to file, in addition to stdout, by setting the following in your Crawler YAML config:
 
 ```
     system_logs_to_file: true
 ```
 
-Crawler also is capable of generating 'event' logs that contain details regarding each crawl being performed. These are meant for debugging and are quite verbose, and as such are only available to be written to file. You can enable these via the following:
+Crawler also is capable of generating 'event' logs that contain details regarding each crawl being performed. These are meant for debugging and are quite verbose, and as such are only available to be written to file (see "Event Logs" section below for more information). You can enable these via the following:
```

Comment:
The .md docs should have a newline per sentence, so that they display better in IDEs. Also, we can directly link to other categories within the page. I think putting the link at the beginning of the paragraph feels neater, too.

```suggestion
Crawler also is capable of generating [event logs](#event-logs) that contain details regarding each crawl being performed.
These are meant for debugging and are quite verbose, and as such are only available to be written to file.
You can enable these via the following:
```

Reply:
Can you handle the other linebreaks throughout the doc? Better than me adding a change suggestion to each 😅